docs/versioned_docs/version-v0.23.0/tutorials/noirjs_app.md

```diff
@@ -99,6 +99,24 @@ At this point in the tutorial, your folder structure should look like this:
 
 #### Some cleanup
 
+Add a `vite.config.js` file containing the following:
+
+```js
+import { defineConfig } from 'vite';
+import topLevelAwait from "vite-plugin-top-level-await";
+
+export default defineConfig({
+    plugins: [
+        topLevelAwait({
+            // The export name of top-level await promise for each chunk module
```

Comment:
I would just remove these comments, they're too descriptive for a tutorial and we don't want for readers to lose focus. Maybe one day we don't need this file at all 🤞